index.d.ts

```diff
@@ -131,4 +131,5 @@ declare namespace EventEmitter {
   export const EventEmitter: EventEmitterStatic;
 }
 
-export = EventEmitter;
+export const EventEmitter;
```

Comment:
This does not seem correct. Also, I think that with this change, the module can no longer be imported using the CommonJS-style:

```js
import EventEmitter = require('eventemitter3');
```